lib/src/screens/wallet_keys/wallet_keys_page.dart

```diff
@@ -57,10 +58,7 @@ class WalletKeysPage extends BasePage {
                       height: 1,
                       padding: EdgeInsets.only(left: 24),
                       color: Theme.of(context).accentTextTheme!.headline6!.backgroundColor!,
-                      child: Container(
-                        height: 1,
-                        color: Theme.of(context).dividerColor,
-                      ),
+                      child: SectionDivider(),
```

Comment:
**Minor:**

since this widget has a const constructor, it would be a bit better for the performance to use the same saved instance by adding `const` before the constructor anywhere its used in